pkgs/development/tools/glslviewer/default.nix

```diff
@@ -1,53 +1,41 @@
-{ lib, stdenv, fetchFromGitHub, glfw, pkg-config, libXrandr, libXdamage
-, libXext, libXrender, libXinerama, libXcursor, libXxf86vm, libXi
-, libX11, libGLU, python3Packages, ensureNewerSourcesForZipFilesHook
-, Cocoa
+{
+    withFFMPG ? true,
+    stdenv, cmake, ninja, lib, fetchFromGitHub,
+    libX11, libXrandr, libXinerama, libXcursor, libXi, libXext, libGLU, ffmpeg,
+    Cocoa
 }:
 
 stdenv.mkDerivation rec {
   pname = "glslviewer";
-  version = "1.6.8";
-
+  version = "2.0.4";
   src = fetchFromGitHub {
-    owner = "patriciogonzalezvivo";
-    repo = "glslViewer";
-    rev = version;
-    sha256 = "0v7x93b61ama0gmzlx1zc56jgi7bvzsfvbkfl82xzwf2h5g1zni7";
+      owner = "patriciogonzalezvivo";
+      repo = "glslViewer";
+      fetchSubmodules = true;
```

Comment:
```suggestion
```
Everything is packages in nixpkgs.

Reply:
Is it? where are `minuaudio` and `ada` (the lib not the compiler with the same name)?

Reply:
oh, I missed some things. I t would be better to package those, too instead of vendoring them.